foxpuppet/windows/browser/notifications/addons.py

```diff
@@ -39,12 +46,22 @@ def addon_name(self):
     def cancel(self):
         """Cancel add-on install."""
         with self.selenium.context(self.selenium.CONTEXT_CHROME):
-            self.root.find_element(*self._cancel_locator).click()
+            if self.window.firefox_version >= LooseVersion('53'):
+                # Firefox >= 53
```

Comment:
Could these comments be a bit more descriptive? Just thinking from the documentation perspective.